src/util/typography.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * Standardized typography styling.
+ *
+ * @packageDocumentation
```

Comment:
Did not know that this was used to help documentation parsers use the comment text to summarize the file. Cool!!!!!